EssentialsGeoIP/src/com/earth2me/essentials/geoip/EssentialsGeoIPPlayerListener.java

```diff
@@ -72,7 +72,7 @@ public void delayedJoin(Player player) {
                     String city;
                     String region;
                     String country;
-                    if (config.getBoolean("enable-locale", true)) {
+                    if (config.getBoolean("enable-locale")) {
```

Comment:
Why is this done? This will generate an NPE if `enable-locale` is not set.

Reply:
It is always set. Check [line 123](https://github.com/EssentialsX/Essentials/pull/1784/files/d9d212d055495d5b4e14ffc28d7196d378638a32#diff-240daa1d27cf9f4b28ad7adc68e81fd5R123).